Accept Ctrl-p and Ctrl-n as up and down by default (#180)
mikaelmello/inquire: CHANGELOG.md

- Add strict clippy lints to improve code consistency and readability.
- Expand workflow clippy task to lint all-features in workspace.
- Add docs badge to readme.

> - Accept Ctrl-p and Ctrl-n as up and down by default in select, multi-select.

Reviewer: ```suggestion
- Keybindings Ctrl-p and Ctrl-n added for Up and Down actions
```

nit
